__tests__/component-test.js

```diff
@@ -0,0 +1,38 @@
+jest.dontMock('../module/');
+
+import React from 'react/addons';
+import { equal } from 'assert';
+
+const { TestUtils } = React.addons;
+const {
+	Cond,
+	Case,
+	Default
+} = require('../module/');
+describe('React-Cond components', () => {
+
+  it('should render the child component if the condition value is equal to `props.value`', () => {
+		let component = TestUtils.renderIntoDocument(
+			<Cond value={1}>
+				<Case test={2}><h1>unexpected</h1></Case>
+				<Case test={1}><h1>expected</h1></Case>
+				<Default><h1>default</h1></Default>
+			</Cond>
+		);
+		let val = TestUtils.findRenderedDOMComponentWithTag(component, 'h1');
+		equal(val.getDOMNode().textContent, 'expected');
+	});
+
+  it('should render the default value if no cases match', () => {
+		let component = TestUtils.renderIntoDocument(
+			<Cond value={10}>
+				<Case test={2}><h1>unexpected</h1></Case>
+				<Case test={1}><h1>expected</h1></Case>
```

Comment:
Yeah, I was also thinking that they could be easily mixed but I don't think there is any reasonable use case for it.